backend/src/embedding/local-embbeding-provider.ts

```
@@ -0,0 +1,27 @@
+      }
+    } catch (error) {
+      this.logger.log(`error when using ${model} api`);
+    }
+  }
+
+  static async getEmbList() {
+    Object.values(EmbeddingModel).forEach((model) => {
+      this.logger.log(model);
+    });
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)